app/src/main/java/at/roteskreuz/stopcorona/screens/questionnaire/QuestionnaireFragment.kt

```diff
@@ -133,6 +124,17 @@ class QuestionnaireFragment : BaseFragment(R.layout.fragment_questionnaire) {
                     Decision.SELFMONITORING -> startQuestionnaireSelfMonitoringFragment()
                 }
             }
+
+        viewModel.getQuestionnaireWithAnswers(
```

Comment:
If you are not observing new data anymore, the fetch on start is useless.

Reply:
done